app/models/jam.rb

```diff
@@ -33,6 +35,12 @@ def duration
 
   private
 
+  def make_midi_solo
+    if ['audio/midi', 'audio/x-midi'].include?(file.content_type)
```

Comment:
since we have a helper too, prob makes sense to expose `midi?` on `jam` instead so there would be no code duplication